### ISSUE TITLE
Unable to Navigate Submission Form Using Keyboard Only

### ISSUE THREAD

**Issue by klamanu** [DSpace/dspace-angular#4584]
I would like to inform you that I am unable to navigate the submission form in DSpace 8.1 using only the keyboard (Tab and arrow keys). I tested this on the demo site, which is currently running version 9, and encountered the same issue. I tried using Firefox, Chrome, and Edge.

For example:

* Without using the mouse, I cannot change the collection of the item in submission using only the keyboard.

* The Tab key does not allow navigation beyond the date field, and it is not possible to enter the year in this field without using the mouse.

* Additionally, for dropdown fields that do not have a default value selected (such as language and type), it is not possible to choose a value using only the keyboard.

https://groups.google.com/g/dspace-tech/c/UqdhY5eWLHk/m/gfRWATDvCQAJ?utm_medium=email&utm_source=footer

**Comment by lgeggleston** [comment]
Just added this ticket as a sub-issue: https://github.com/DSpace/dspace-angular/issues/6090
It has some additional detail, but overall includes just a subset of the other issues with the keyboard navigation of the submission form.

**Comment by abelc-escire** [comment]
Hi @lgeggleston, I hope you are doing great! could you please assign me the task?

Regards!
